### DISCUSSION TITLE
@vsevjednom-cz Management of forked code

### DISCUSSION THREAD

**Discussion by dennisguse** [OpenTracksApp/OpenTracks#1308] [Q&A]
https://github.com/OpenTracksApp/OpenTracks/pull/1293#issuecomment-1169884376

Your comment gave my an itch - how do you actually manage your fork?
Best guess: it is not a Git repo, right?
So, how do you sync with the upstream repo?

If you don't use Git for this: why?
And what do you need to switch to Git?

**Comment by dennisguse** [comment]
@vsevjednom-cz ?

**Reply by dennisguse** [reply]
@vsevjednom-cz ?

**Comment by vsevjednom-cz** [comment]
You're right. 😍

**Reply by dennisguse** [reply]
My suggestion would then be to put in the work to handle this via Git:
* fork the OpenTracks repo (you already did this)
* every now and then  sync the forks: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-command-line
* commit and push your changes (might be a new branch)

if you want to get the upstream changes, you can use `git rebase`.
And you will also see what breaks :)
